fr.handbrake.ghb.json

```diff
@@ -0,0 +1,147 @@
+{
+    "app-id": "fr.handbrake.ghb", 
+    "modules": [
+        {
+            "name": "handbrake", 
+            "modules": [
+                {
+                    "sources": [
+                        {
+                            "url": "http://www.nasm.us/pub/nasm/releasebuilds/2.13.02/nasm-2.13.02.tar.xz", 
+                            "sha256": "8ac3235f49a6838ff7a8d7ef7c19a4430d0deecc0c2d3e3e237b5e9f53291757", 
+                            "type": "archive"
+                        }
+                    ], 
+                    "cleanup": [
+                        "*"
+                    ], 
+                    "name": "nasm"
+                }
+            ], 
+            "config-opts": [
+                "--flatpak", 
+                "--disable-gtk-update-checks"
+            ], 
+            "sources": [
+                {
+                    "url": "https://download.handbrake.fr/releases/1.1.1/HandBrake-1.1.1-source.tar.bz2", 
+                    "sha256": "e3390c5fd901fb06d72e29c62a63d373d5fb5b3467295d114d815ae7b78a9d7a", 
+                    "type": "archive", 
+                    "strip-components": 1
+                }, 
+                {
+                    "commands": [
+                        "mkdir -p download"
```

Comment:
Use the `dest` property on sources and you can then avoid this. (It is kinda undefined that paths work in `dest-filename`).

Reply:
It didn't look like dest-filename would automatically create the destination directory.  Does it?

Reply:
No that is why I'm saying to use `dest` for the subdir in combination with `dest-filename` which is only the filename.